### ISSUE TITLE
Embedding igv.js-jupyter into a notebook

### ISSUE THREAD

**Issue by sooheelee** [DataBiosphere/leonardo#780]
Please, can someone weigh in on https://github.com/igvteam/igv.js-jupyter/issues/23?

We are trying to figure out how to get igv.js-jupyter running on Leonardo. I need this functionality to visualize BigWig files from the GATK gCNV workflow.

**Comment by sooheelee** [comment]
@rtitle is looking into this.

**Comment by sharmatime** [comment]
Hey Soo Hee - Thanks for bringing this to our attention. I wasn't aware you would run IGV.js in a notebook so it's really interesting. Someone from the interactive analysis team will have to investigate this more deeply, but we're currently a small team that's working on a backlog of priorities. Can you give me more context around your request? Is this for the BroadE workshop, or is this for something else, urgency, blocking, etc.

**Comment by sooheelee** [comment]
Thanks, @sharmatime for the response. I'm unblocked in my individual work as I've been given permission to take my secondary analyses off of FireCloud. I'm running igv.js-jupyter in a Jupyter notebook on my laptop.

For the Comms team, and especially for the upcoming BroadE workshop, where as far as I'm aware my teammates will be adapting each GATK hands-on tutorial to run on FireCloud, it would be good for them to have the ability to showcase genomic data transformations _within_ the Jupyter notebook, i.e. with the igv.js-jupyter extension, and for the audience members to be able to do so on their own laptops. Running the igv-jupyter extension is especially applicable for our newer hands-on tutorials, especially the Mutect2 tutorial, because the data I used in developing these are aligned to hg38 and being able to interact with the data live is essential to some of the take home messages of the tutorials. Furthermore, last I checked, FireCloud's _Analysis_ tab does NOT support hg38. It only supports hg19/b37. 

@rcmajovski should actually weigh in here as they are actually in charge of adapting the materials for the upcoming BroadE workshop and I am not involved.
